Woof/Woof/View/DetailedPetSitterView/ContactDelailsSectionView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+/// A view displaying a section for the detailed sitter's information.
+struct ContactDelailsSectionView: View {
+    /// The name of an icon.
+    let image: String
+    /// The informational text to be displayed.
+    let message: String
+
+    var body: some View {
+        RoundedRectangle(cornerRadius: AppStyle.UIElementConstant.cornerRadius)
+            .padding(.horizontal)
+            .frame(height: AppStyle.UIElementConstant.frameContactDetailsSectionHeight)
```

Comment:
Do we need to constrain the height of the element?

Reply:
I removed the element's height from this reusable view.